job-dsls/jobs/prod/prod_cron_meta_nightly_pipeline.groovy

```diff
@@ -57,13 +57,13 @@ def RHBOP_CURRENT_DROOLS_VERSION='8.29.0'
 // def DROOLS_CURRENT_PRODUCT_VERSION= OPTAPLANNER_CURRENT_PRODUCT_VERSION
 
 // Drools Ansible Integration
+def DAI_NEXT_PRODUCT_VERSION='' // took from project pom
+def DAI_NEXT_DROOLS_VERSION='999.0.0'
```

Comment:
This is now required because the Drools version is now 999-SNAPSHOT (without minor and micro) and before it used to be something like 8.43.0-SNAPSHOT, so we need this otherwise PME is not able to properly deal with the versions in the nightly.